packages/node-protocol/README.md

```diff
@@ -0,0 +1,20 @@
+## Replay Node Protocol
```

Comment:
I think moving this out to its own package makes it clearer why it's needed and also easier to re-use if we decide to publish it down the line

Reply:
Yeah, that seems like a good call.